cogs/helpers.py

```diff
@@ -116,6 +116,7 @@ def retrieve_string(label):
         # and more (or the normal temperature if it is higher), and a linear
         # roll-off of the wind chill between 10°C and 14°C
         # (https://blog.metservice.com/FeelsLikeTemp)
+        # Written by @le-potate
```

Comment:
create a new branch for authorship fixup

Reply:
dunno if this matters enough, rewriting history is aggravating once it's pushed

Reply:
agree with david, i'll be more careful next time tho